.travis.yml

```diff
@@ -1,16 +1,29 @@
+sudo: required
+
 language: c
 
+services:
+    - docker
+
 compiler: gcc
 
 before_install:
  - sudo apt-get update -qq
- - sudo apt-get install -qq libtalloc-dev uthash-dev libarchive-dev gdb strace realpath
- - sudo pip install cpp-coveralls
+ - sudo apt-get install -qq libtalloc-dev libarchive-dev uthash-dev gdb strace realpath
+ - sudo -H pip install cpp-coveralls==0.3.12
 
-script: if [ ${COVERITY_SCAN_BRANCH} != 1 ]; then make -C src loader.exe loader-m32.exe build.h && env CFLAGS=--coverage LDFLAGS='--coverage' make -C src proot && env PATH=/bin:/usr/bin:/sbin:/usr/sbin:$PWD/src make -C tests; fi
+script: make -C src loader.exe loader-m32.exe build.h && env CFLAGS=--coverage LDFLAGS='--coverage' make -C src proot care && env PATH=/bin:/usr/bin:/sbin:/usr/sbin:$PWD/src PROOT_NO_SECCOMP=1 make -C tests
```

Comment:
The Seccomp mode should be enabled for the tests.

Reply:
yes debug until we merge #130 ;)